Sources/ManageSettings.php

```diff
@@ -179,9 +179,10 @@ function ModifyBasicSettings($return_config = false)
 			'select',
 			'jquery_source',
 			array(
-				'auto' => $txt['jquery_auto'],
+				'google_cdn' => $txt['jquery_google_cdn'],
```

Comment:
Please change 'google_cdn' back to just 'cdn'.

Reply:
See comment for Load.php